[2024-12-16 16:11:51,150][zerorpc.core][ERROR] - 
Traceback (most recent call last):
  File "/home/my/anaconda3/envs/m3p2i-aip/lib/python3.8/site-packages/zerorpc/core.py", line 153, in _async_task
    functor.pattern.process_call(self._context, bufchan, event, functor)
  File "/home/my/anaconda3/envs/m3p2i-aip/lib/python3.8/site-packages/zerorpc/patterns.py", line 30, in process_call
    result = functor(*req_event.args)
  File "/home/my/anaconda3/envs/m3p2i-aip/lib/python3.8/site-packages/zerorpc/decorators.py", line 44, in __call__
    return self._functor(*args, **kargs)
  File "reactive_tamp.py", line 125, in run_tamp
    self.motion_planner.command(self.sim._dof_state[0])[0]
  File "/home/my/m3p2i-aip-master/src/m3p2i_aip/planners/motion_planner/mppi.py", line 245, in command
    cost_total = self._compute_total_cost_batch_halton()
  File "/home/my/m3p2i-aip-master/src/m3p2i_aip/planners/motion_planner/mppi.py", line 419, in _compute_total_cost_batch_halton
    self.cost_total, self.states, self.actions, self.ee_states = self._compute_rollout_costs(self.perturbed_action)
  File "/home/my/m3p2i-aip-master/src/m3p2i_aip/planners/motion_planner/mppi.py", line 306, in _compute_rollout_costs
    c = self._running_cost(state) # every time stes you get nsamples cost, we need that as output for the discount factor
  File "/home/my/m3p2i-aip-master/src/m3p2i_aip/planners/motion_planner/mppi.py", line 210, in _running_cost
    return self.running_cost(state)
  File "reactive_tamp.py", line 138, in running_cost
    return self.objective.compute_cost(self.sim)
  File "/home/my/m3p2i-aip-master/src/m3p2i_aip/planners/motion_planner/cost_functions1.py", line 84, in compute_cost
    get_motion_cost_1 = self.get_motion_cost_1(sim)
  File "/home/my/m3p2i-aip-master/src/m3p2i_aip/planners/motion_planner/cost_functions1.py", line 264, in get_motion_cost_1
    distance, self.nn_grad = self.distance_repulsion_nn(sim, q_prev, aot=False)
  File "/home/my/m3p2i-aip-master/src/m3p2i_aip/planners/motion_planner/cost_functions1.py", line 444, in distance_repulsion_nn
    nn_dist, nn_grad, nn_minidx = self.nn_model.dist_grad_closest(nn_input_padded[:, 0:-1])
  File "/home/my/m3p2i-aip-master/src/m3p2i_aip/planners/mlp_learn/sdf/robot_sdf.py", line 146, in dist_grad_closest
    self.grads[:n_inputs, :, 0] = q.grad  # fill in one column of Jacobian
RuntimeError: The expanded size of the tensor (10) must match the existing size (30) at non-singleton dimension 1.  Target sizes: [82, 10].  Tensor sizes: [82, 30]
